src/frontend/hooks/server/media.ts

```diff
@@ -6,6 +6,8 @@ import {URL} from 'react-native-url-polyfill';
 
 import {DraftPhoto} from '../../contexts/PhotoPromiseContext/types';
 import {useProject} from './projects';
+import {Buffer} from 'buffer';
```

Comment:
remove unused import

Reply:
It is used below, without this import the app crashes.

Reply:
My bad, it was moved to another file eventually. Will correct.